src/main/java/com/letsparty/web/controller/LetsPartyController.java

```diff
@@ -63,6 +77,35 @@ public String post (@Valid LetsPartyPostForm letsPartyPostForm, BindingResult er
 	    return "redirect:/letsparty";
 	}
 	
+	// 렛츠파티 리스트 표시
+	@GetMapping("search")
```

Comment:
```suggestion
	@GetMapping("/search")
```
동작은 똑같지만 우리 코드스타일에 `/` 표기가 적합한 것 같아요.